doc/source/release-notes/Percona-XtraDB-Cluster-5.6.47-28.40.rst

```diff
@@ -0,0 +1,24 @@
+.. _PXC-5.6.47-28.40:
+
+================================================================================
+*Percona XtraDB Cluster* 5.6.47-28.40
+================================================================================
+
+:Date: May 4, 2020
+:Installation: `Installing Percona XtraDB Cluster <https://www.percona.com/doc/percona-xtradb-cluster/5.6/installation.html>`_
+
+Improvements
+================================================================================
+
+* :jirabug:`PXC-3006`: Added information about xinetd 'cps' option in PXC HAProxy documentation
```

Comment:
PXC-3006 is in review in jira, and affects only 5.7

Reply:
Removed 3006